README.md

```diff
@@ -17,15 +17,104 @@ This project ensures that the Kubernetes control plane responds appropriately to
 - Webhook feature to send shutdown or restart notification messages
 - Unit & Integration Tests
 
+## Differences from v1
+
+The first major version of AWS Node Termination Handler (NTH) originally operated only as a daemon deployed to every desired node in the cluster (aka IMDS Mode); later the option to deploy only a single pod which read events from an SQS queue was added (aka Queue Processor Mode). Both heavily utilized Helm for configuration, and changing configuration meant updating the deployment.
+
+This second major version of NTH aims to refine the Queue Processor Mode. Only a single pod is deployed and configuration is done using a new custom resource called *Terminators*. A *Terminator* contains much of the configuration about where NTH should fetch events, what actions to take for a given event type, filter nodes to act upon, and webhook notifications. Multiple *Terminators* may be deployed, modified, or removed without needing to redeploy NTH itself.
+
 ## Getting Started
 
-### Infrastructure Setup
+### 1. Setup Infrastructure
 
-TBD
+#### 1.1. Create an IAM OIDC Provider
+
+Your EKS cluster must have an IAM OIDC Provider. Follow the steps in [Create an IAM OIDC provider for your cluster](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) to determine whether your EKS cluster already has an IAM OIDC Provider and, if necessary, create one.
+
+#### 1.2. Create the NTH Service Account
+
+##### 1.2.1. Create the IAM Policy
+
+Download the service account policy template for AWS CloudFormation at https://github.com/aws/aws-node-termination-handler/releases/download/VERSION/infrastructure.yaml
```

Comment:
I assume this means that https://github.com/aws/aws-node-termination-handler/blob/v2/resources/infrastructure.yaml will be one of the release artifacts?

Not sure how the release process will work in `v2`, but in `main`, our release script [does a regex replace](https://github.com/aws/aws-node-termination-handler/blob/main/scripts/prepare-for-release#L166-L198) in README and a few other files to allow us to put real version numbers in instructions like this and keep them up to date. If `v2` has a similar system, it would streamline things slightly for the user. If not, I'd recommend at least putting `VERSION` inside angle brackets, like you do below for values the user must supply themselves.